src/components/DigitalCredentials/IssueCredentials.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <section id="issue-credentials" class="py-8">
+
+    <article id="issue-credentials-header">
+      <v-row no-gutters>
+        <v-col cols="12" lg="11">
+          <h3>Issue Credential to Digital Wallet</h3>
+          <p class="mt-4 font-14">
+            <span v-if="hasRegisteredWallet">
+              Issue the business credentials to the registered digital wallet. You can choose to issue two kinds of
+              Digital Credentials. Having both credentials will help with accessing services quicker. Credentials are
+              issued by sending Credential Offers to the digital wallet app. Once the Credential Offers are sent, the
+              digital wallet app on your phone will ask you to claim the Credential Offers. When the Credential Offers
+              are claimed, the credential is stored on the app.
+            </span>
+            <span v-else>
+              There are no digital wallets registered with this business. Please go back to the previous step and
+              register a digital wallet.
+            </span>
+          </p>
+        </v-col>
+      </v-row>
+    </article>
+
+    <article v-if="hasRegisteredWallet">
+      <v-row no-gutters>
+        <v-col cols="12" lg="12">
+          <v-card flat rounded class="px-2 mt-3">
+            <v-card-text class="px-6 py-8">
+
+              <!-- Credential Offer -->
+              <v-row no-gutters>
+                <v-col cols="12" lg="8" class="ma-auto">
+                  <h4>Business Digital Credential</h4>
+                </v-col>
+                <v-col cols="12" lg="4" class="pr-2">
+                  <v-btn
+                    v-if="issuedCredentials.length === 0"
+                    id="next-btn"
+                    large color="primary"
+                    class="font-weight-bold px-6 float-right"
+                    :loading="isSendingOffer"
+                    @click="sendCredentialOffer(DigitalCredentialTypes.BUSINESS)"
+                  >Send Credential Offer</v-btn>
+
+                  <span v-else-if="isPendingAcceptedOffer" class="font-weight-bold alert-text">
```

Comment:
Should this be a div with the text on line 48 inside a span?

Same for line 51.

Reply:
Same for line 70.

Reply:
It could be for sure, i guess the styling here only applies to the text anyways, so make sense

Reply:
My rationale: Divs are block elements while spans are inline elements. Spans inside divs are typical. Spans inside spans is weird :) I think it's better to use divs where possible. 

Ref: https://www.w3schools.com/html/html_blocks.asp

Reply:
Sounds good to me, makes perfect sense! Updated

Reply:
If this messed up your layout, since the class is now in the class, you can use template elements for the v-ifs etc.

Reply:
All good! The div is conditional, so all the same.